src/troute-routing/troute/routing/diffusive_utils.py

```diff
@@ -541,15 +625,17 @@ def diffusive_input_data_v02(
     -------
     diff_ins -- (dict) formatted inputs for diffusive wave model
     """
-    
+
     # lateral inflow timestep (sec)
     dt_ql_g = dt * qts_subdivisions
     # upstream boundary condition timestep (sec)
     dt_ub_g = dt
     # downstream boundary condition timestep (sec)
-    dt_db_g = dt * qts_subdivisions
+    dt_db_g = 360.0 # dt * qts_subdivisions
```

Comment:
hard-wiring values is risky. What happens if this changes? Also, the downstream boundary data has a 6-minute time interval? That seems quite arbitrary.